cmd/influx/cli/cli.go

```diff
@@ -322,8 +322,41 @@ func (c *CommandLine) use(cmd string) {
 		return
 	}
 	d := args[1]
-	c.Database = d
-	fmt.Printf("Using database %s\n", d)
+
+	// validate if specified database exists
+	query := "SHOW DATABASES"
```

Comment:
Very minor, but just shove `SHOW DATABASES` inside the call to `Query`. A new variable isn't really needed, don't you agree?